src/constants.ts

```diff
@@ -14,6 +14,7 @@ export const CHAIN_IDs = {
   OPTIMISM_GOERLI: 420,
   ZK_SYNC_GOERLI: 280,
   BASE_GOERLI: 84531,
+  BASE_SEPOLIA: 111551111,
```

Comment:
ooc, have you tested this? 84532 is the chain ID reported by multiple sources.

fwiw I've also raised a PR on top of this one to add some extra chainIds, but it's nice to inherit the token definitions from this one: https://github.com/across-protocol/constants-v2/pull/13.